shark/shark/api/shark.api

```diff
@@ -26,6 +26,7 @@ public abstract class shark/AndroidReferenceReaders : java/lang/Enum, shark/Chai
 	public static final field MESSAGE_QUEUE Lshark/AndroidReferenceReaders;
 	public static final field SAFE_ITERABLE_MAP Lshark/AndroidReferenceReaders;
 	public synthetic fun <init> (Ljava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public static fun getEntries ()Lkotlin/enums/EnumEntries;
```

Comment:
What's up with this new API showing up everywhere?

Reply:
https://kotlinlang.org/docs/whatsnew19.html#stable-replacement-of-the-enum-class-values-function